Streetcode/Streetcode.BLL/MediatR/Streetcode/RelatedFigure/GetByStreetcodeId/GetRelatedFiguresByStreetcodeIdHandler.cs

```diff
@@ -47,12 +47,19 @@ public async Task<Result<IEnumerable<RelatedFigureDTO>>> Handle(GetRelatedFigure
 
     private IQueryable<int> GetRelatedFigureIdsByStreetcodeId(int StreetcodeId)
     {
-        var observerIds = _repositoryWrapper.RelatedFigureRepository
+        try
+        {
+            var observerIds = _repositoryWrapper.RelatedFigureRepository
             .FindAll(f => f.TargetId == StreetcodeId).Select(o => o.ObserverId);
 
-        var targetIds = _repositoryWrapper.RelatedFigureRepository
-            .FindAll(f => f.ObserverId == StreetcodeId).Select(t => t.TargetId);
+            var targetIds = _repositoryWrapper.RelatedFigureRepository
+                .FindAll(f => f.ObserverId == StreetcodeId).Select(t => t.TargetId);
 
-        return observerIds.Union(targetIds).Distinct();
+            return observerIds.Union(targetIds).Distinct();
+        }
+        catch (ArgumentNullException)
+        {
+            return null;
+        }
```

Comment:
If we catch it, we should do something with exception

Reply:
refactored this code a little, to remove this null